.browserslistrc

```diff
@@ -1 +0,0 @@
-extends @ebay/browserslist-config
```

Comment:
This wasn't getting picked up?

Reply:
It was getting picked up everywhere which were causing the other modules aside from abort-group to target those browsers. Not sure if it was the case before upgrading babel though.